deployments/with-creds/ci/values.yaml

```diff
@@ -82,11 +82,20 @@ concourse:
     web:
       auth:
         mainTeam:
+          config: |
+            roles:
+            - name: pipeline-operator
+              github:
+                teams: ["concourse:contributor"]
           localUser: admin
           github:
             team: concourse:Pivotal
         github:
           enabled: true
+      configRBAC: |
+        pipeline-operator:
+        - RerunJobBuild
+        - CheckResource
```

Comment:
I'm pretty sure this is a no-op, since [these](https://github.com/concourse/concourse/blob/cb9530ecf62340c4fa5c17e14ec2f1a72d0c6b0d/atc/api/accessor/roles.go#L29) [actions](https://github.com/concourse/concourse/blob/cb9530ecf62340c4fa5c17e14ec2f1a72d0c6b0d/atc/api/accessor/roles.go#L48) are already assigned to the pipeline-operator role.

If you want to restrict pipeline operators so that they can _only_ perform these actions (like no pinning/pausing/aborting/enabling/disabling) I think you want to promote those actions to `member`:
```suggestion
      configRBAC: |
        member:
        - AbortBuild
        - CreateJobBuild
        - PauseJob
        - UnpauseJob
        - ClearTaskCache
        - UnpinResource
        - SetPinCommentOnResource
        - CheckResourceWebHook
        - CheckResourceType
        - EnableResourceVersion
        - DisableResourceVersion
        - PinResourceVersion
        - PausePipeline
        - UnpausePipeline
```
or else you could grant these github team members the `viewer` role and demote rerunning and checking so that viewers can do it:
```suggestion
      configRBAC: |
        viewer:
        - RerunJobBuild
        - CheckResource
```